examples/readme.md

```diff
@@ -0,0 +1,41 @@
+Logsink examples
+================
+
+Examples to create logsinks on Google Cloud Platform using the `logsink` module.
+
+Variables
+---------
+The Terraform variables for all the examples are defined in the `variables.tfvars` in the `examples/`
+directory. Replace the variables by your own before running the examples.
+
+Examples
+--------
+Examples are structured as follows:
+
+* ***base/*** creates a test folder identified by the variable `folder_name` and a test project identified by the variable `project_name`. This example **only** creates the structure required to run the other examples.
+Skip running this if you already have a test folder and a test project created.
+
+* ***pubsub/*** creates 3 sinks (org-level, folder-level and project-level) with a Cloud Pub/Sub topic as the destination.
```

Comment:
Instead of organizing by destination I would suggest organizing by *source* for two reasons:

- It's the more likely place people start from (I want to export logs from my org/folder/project) before they decide the destination.
- It makes it possible to work with the examples even if I only have project-level access, for example.

So I'd suggest 3 example folders (org, folder, project) which each include an example of each destination.

Reply:
Right, since that's considerable refactoring, can we keep this as an [issue](https://github.com/terraform-google-modules/terraform-google-log-export/issues/5) for now ?

Reply:
SGTM.